### ISSUE TITLE
Can I have some help downloading .minerva

### ISSUE THREAD

**Issue by SniperXKiller** [gmariani/minerva#91]
Hi, I TRIED to download minerva, but when I clicked the link to download minerva ( https://mariani.life/projects/minerva/ ) it says 
"Notice
It appears your browser does not support:

Application Cache API - Used to take the application offline
Please upgrade your browser in order to use .minerva."
I am using Chrome 86 on Windows 10. CAN I HAVE HELP?

**Comment by SniperXKiller** [comment]
> Hi, I TRIED to download minerva, but when I clicked the link, it says
> "Notice
> It appears your browser does not support:
> 
> Application Cache API - Used to take the application offline
> Please upgrade your browser in order to use .minerva."
> I am using Chrome 86 on Windows 10. CAN I HAVE HELP?

**Comment by Zaseth** [comment]
I just tested it, same thing.

**Comment by gmariani** [comment]
Just hit ok to close the notice. Appears they changed something in Chrome, I'll have to update it to check correctly. You cannot "download" minerva anymore after I dropped Adobe AIR. It's still just a website. All that does it allow it to work offline, but if the notice is a false positive it should still work.
I tested on Chrome 86 by simulating a disconnected connection and it still loaded from cache. So i'd say just ignore the warning if you're using a current version of Chrome.
